lib/rules/max-empty-lines/index.js

```diff
@@ -98,6 +159,39 @@ const rule = function(max, options) {
         }
       }
     }
+
+    function replaceEmptyLines(max, str, isSpecialCase = false) {
+      const repeatTimes = isSpecialCase ? max : max + 1;
+
+      if (repeatTimes === 0) {
+        return "";
+      }
+
+      const emptyLFLines = _.repeat("\n", repeatTimes);
+      const emptyCRLFLines = _.repeat("\r\n", repeatTimes);
+
+      let result;
+
+      if (/(\r\n)+/g.test(str)) {
```

Comment:
May be it will be better to construct regexp on fly for example:
```
result = str
.replace(new RegExp(`[\r\n]{${repeatTimes},}`, 'g'), $1 => $1.substr(0, repeatTimes * 2))
.replace(new RegExp(`\n{${repeatTimes},}`, 'g'), $1 => $1.substr(0, repeatTimes))
```
but anyway I hope noone will use 'lf' and 'crlf' in one file